README.md

```diff
@@ -253,6 +253,9 @@ Visit [bentoml/gallery](https://github.com/bentoml/gallery) repository for more
 ### FastText
 * Text Classification - [Google Colab](https://colab.research.google.com/github/bentoml/gallery/blob/master/fasttext/text-classification/text-classification.ipynb) | [nbviewer](https://nbviewer.jupyter.org/github/bentoml/gallery/blob/master/fasttext/text-classification/text-classification.ipynb) | [source](https://github.com/bentoml/gallery/blob/master/fasttext/text-classification/text-classification.ipynb)
 
+### ONNX
```

Comment:
also add the examples page in the docs?

Reply:
yep. Also add the missing the fasttext example